### ISSUE TITLE
error in "Exporting and Archiving" section of docs

### ISSUE THREAD

**Issue by ralexx** [ioam/holoviews#1743]
Clicking http://holoviews.org/user_guide/Exporting_and_Archiving.html returns:
```
Notebook conversion failed with the following traceback: Cell raised uncaught exception: [0;31m---------------------------------------------------------------------------[0m [0;31mNameError[0m Traceback (most recent call last) [0;32m [0m in [0;36m [0;34m()[0m [0;32m----> 1[0;31m [0mpenguins[0m[0;34m[[0m[0;34m:[0m[0;34m,[0m[0;34m:[0m[0;34m,[0m[0;34m'R'[0m[0;34m][0m[0;34m.[0m[0mrelabel[0m[0;34m([0m[0;34m"Red"[0m[0;34m)[0m [0;34m+[0m [0mpenguins[0m[0;34m[[0m[0;34m:[0m[0;34m,[0m[0;34m:[0m[0;34m,[0m[0;34m'G'[0m[0;34m][0m[0;34m.[0m[0mrelabel[0m[0;34m([0m[0;34m"Green"[0m[0;34m)[0m [0;34m+[0m [0mpenguins[0m[0;34m[[0m[0;34m:[0m[0;34m,[0m[0;34m:[0m[0;34m,[0m[0;34m'B'[0m[0;34m][0m[0;34m.[0m[0mrelabel[0m[0;34m([0m[0;34m"Blue"[0m[0;34m)[0m[0;34m[0m[0m [0m [0;31mNameError[0m: name 'penguins' is not defined 
```

**Comment by jlstevens** [comment]
Thanks for reporting this! This will be fixed for 1.8.2.

**Comment by philippjfr** [comment]
Appears to be fixed in the most recent dev build: http://dev.holoviews.org/user_guide/Exporting_and_Archiving.html
